.github/workflows/dep-review.yaml

```diff
@@ -0,0 +1,23 @@
+name: Dependency Review
```

Comment:
As noted in conversation, I want to reduce Quamina's dependencies to as close to zero as possible.  For example, the event flattener has its own JSON parser which is insanely faster than Go's default one (which first of all isn't that hard and also Quamina's goals are very different).  Is there anything to do in this section to reflect that design goal?

Reply:
This action first helps to detect newly added dependencies and forces PR creators to keep them up2date.

So I think it's useful for your goal of (close to) 0 deps:

a) flagging/highlighting unwanted deps in PRs
b) as project grows, maintain healthy dependencies